src/App.js

```diff
@@ -1,17 +1,68 @@
 import './App.css';
-import React from 'react';
-
-function App() {
-  // fetches JSON data passed in by flask.render_template and loaded
-  // in public/index.html in the script with id "data"
-  // const args = JSON.parse(document.getElementById("data").text);
-
-  // This main page will be further designed with react components
-  // so the user can interact with the main app page. Some Data will also
-  // be present for viewing when user lands on this page.
-  return (
-    <h1> ! Welcome To Main App Page User !</h1>
-  );
-}
-
-export default App;
+import React from "react";
+import { Dropdown, Option } from "./Dropdown";
+
+/*NOTE:
+  Unaware how to connect fdc.py API calls to search bar function.
+  I will need help trying to figure out how to connectthe search bar 
+  and USDA API to return user requested results. As it stands for an 
+  MVP we have good functionality for the main app page, and we can either
+  add more functionality than what we have now or save some features to add
+  into the app for Sprint 2.
+
+*/
+export default function App() {
+
+  //Will handle the diet components
+  const [optionValue, setOptionValue] = useState("");
+  const handleSelect = (e) => {
+    console.log(e.target.value);
+    setOptionValue(e.target.value);
+  };
+
+  //Will handle the search function
+    const form = useRef(null)
+
+    const submit = e => {
+      e.preventDefault()
+      const data = new FormData(form.current)
+      fetch('https://api.nal.usda.gov/fdc/v1/' + query, { method: 'POST', body: data })
+        .then(res => res.json())
+        .then(data => console.log(data))
+    }
```

Comment:
I know all of this is temporary, so just a note: right now, this will likely throw an error, because `query` is not defined. The API key is also not available on the client.